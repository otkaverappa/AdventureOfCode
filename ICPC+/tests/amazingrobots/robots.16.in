20 20
####################
#X#...#...#...#....#
#.#.#.#.#.#.#.#.#.##
#.#.#.#.#.#.#.#.#..#
#.#.#.#.#.#.#.#.##.#
#.#.#.#.#.#.#.#.#..#
#.#.#.#.#.#.#.#.#.##
#.#.#.#.#.#.#.#.#..#
#.#.#.#.#.#.#.#.##.#
#.#.#.#.#.#.#.#.#..#
#.#.#.#.#.#.#.#.#.##
#..................#
#.#.#.#.#.#.#.#.##.#
#.#.#.#.#.#.#.#.#..#
#.#.#.#.#.#.#.#.#.##
#.#.#.#.#.#.#.#.#..#
#.#.#.#.#.#.#.#.#..#
#.#.#.#.#.#.#.#.#..#
#...#...#...#...#.##
#################.##
9
12 2 2 E
12 4 2 E
12 6 2 E
12 8 2 E
12 10 2 E
12 12 2 E
12 14 2 E
12 16 2 E
18 18 2 N
20 20
####################
#..................#
...................#
#..................#
#..................#
#..................#
#..................#
#..................#
#..................#
#..................#
#..................#
#..................#
#..................#
#..................#
#..................#
#..................#
#..................#
#..................#
#.................X#
####################
10
2 2 2 S
9 7 4 W
19 9 4 W
17 17 4 W
2 10 2 W
3 4 3 W
2 7 4 W
10 12 4 W
14 16 3 W
11 12 2 W
