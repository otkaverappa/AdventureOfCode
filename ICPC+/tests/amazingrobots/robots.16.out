200
E
S
S
S
S
S
S
S
S
S
S
S
S
S
S
S
S
S
E
E
N
N
N
N
N
N
N
N
N
N
N
N
N
N
N
N
N
E
E
S
S
S
S
S
S
S
S
S
S
S
E
S
S
S
S
S
S
E
E
N
N
N
N
N
N
W
N
N
N
N
N
N
N
N
N
W
W
W
W
W
W
W
W
W
W
W
W
W
W
W
W
N
W
N
W
E
E
S
S
S
S
S
S
S
S
S
S
S
S
S
S
S
S
S
E
E
N
N
N
N
N
N
N
N
N
N
N
N
N
N
N
N
N
E
E
S
S
S
S
S
S
S
S
S
S
S
S
S
S
S
S
S
E
E
N
N
N
N
N
N
N
N
N
N
N
N
N
N
N
N
N
E
E
S
S
E
S
S
W
S
S
E
S
S
W
S
S
E
S
S
W
S
S
E
S
S
W
S
S
S
